src/histolab/tile.py

```diff
@@ -168,10 +168,9 @@ def _has_only_some_tissue(self, near_zero_var_threshold: float = 0.1) -> np.bool
             True if the image is composed by only some tissue. False if the tile is
             composed by all tissue or by no tissue at all.
         """
-        filters = FiltersComposition(Tile).tissue_mask_filters
-        tissue_mask = filters(self._image)
-
-        return np.var(tissue_mask) > near_zero_var_threshold
+        np_img = np.array(self._image.convert("L"))
+        np_img = 1.0 * (np_img > 150)
+        return np.var(np_img) > near_zero_var_threshold
```

Comment:
I see that now the implementation is different with respect to what we discussed, a bit of explanation on why this cutoff at 150?